### DISCUSSION TITLE
How to make tabs behave independently of other tabs

### DISCUSSION THREAD

**Discussion by stephannv** [squidfunk/mkdocs-material#5343] [Q&A] [answered]
Hello, I'm using tabs on my docs and I want to make them independent from each other. Is it possible? 

My tabs are all switching together:
https://user-images.githubusercontent.com/3025661/230737283-54d78854-6bc0-4880-a213-59aa544a9c84.mov

My mkdocs.yml
```yaml
  - pymdownx.tabbed:
      alternate_style: true
```

My code:

```markdown
## Section A

Some text

=== "example_page.cr"

    ...

=== "Output"

    ...


Some text

=== "example_page.cr"

    ....

=== "Output"

    ...

```

**Comment by squidfunk** [comment, marked as answer]
You didn't share your whole `mkdocs.yml`, but it looks like you enabled [`content.tabs.link`](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs). Just remove that from `features`.

**Reply by stephannv** [reply]
Thanks @squidfunk for the help. You're right, I added `content.tabs.link` to `theme.features` and I don't know why 😅.
